Sources/Turf/Turf.swift

```diff
@@ -46,3 +46,21 @@ public func mid(_ coord1: CLLocationCoordinate2D, _ coord2: CLLocationCoordinate
     let heading = coord1.direction(to: coord2)
     return coord1.coordinate(at: dist / 2, facing: heading)
 }
+
+/**
+ Takes a coordinate and calculates the circular polygon given a radius in meters,
+ using a defined number of steps for precision.
+
+ - Parameter center: The center coordinate for the circle.
+ - Parameter radius: The radius of the circle, measured in meters.
+ - Parameter steps: The number of vertices the circle will have. Defaults to 64.
+ */
+public func circle(from center: CLLocationCoordinate2D, radius: CLLocationDistance, steps: Int = 64) -> Polygon {
```

Comment:
Consider turning this freestanding function into a Polygon initializer: `Polygon(center:radius:vertices:)`.